src/schema/causality_jwt.js

```diff
@@ -43,16 +48,15 @@ export default {
       // For logged in and...
     } else if (options.role === "PARTICIPANT" && accessToken) {
       return Promise.all([
-        gravity(`sale/${options.sale_id}`),
-        gravity.with(accessToken)("me"),
```

Comment:
It seems like we fetch `/api/v1/me` _just_ for `me._id`, but we support passing in user id as a header. Is this schema used from Prediction? If so, does that app pass along user id (analogous to access token, Force/Eigen pass in access token and user id, etc.).

Reply:
It looks like prediction [passes in the user token](https://github.com/artsy/prediction/blob/754392ea8bc2fbe36728b4f50e81fb11df9217a7/src/utils/server/api.js#L268-L290) only. Can it be parsed to get the user ID without requiring it to be fetched here?

Reply:
No, those are gravity-signed access tokens, which metaphysics can't decode.

Can Prediction get updated to pass in the user id? That would make it more in line with the other apps using Metaphysics (assuming that user id is fetched/accessible on Prediction).

Reply:
Prediction can fetch the user_id but I don't think it actually stores it (only the user token).

Reply:
Ok.

So...what should we do? Can we update Prediction? Should we undo this? Work with me here!

Reply:
I think we should not block this on prediction passing the user ID, so I would remove that and add a TODO/issue somewhere if it's important!

Reply:
Sounds good.